packages/cli/cli/src/commands/generate/generateAPIWorkspaces.ts

```diff
@@ -52,6 +54,23 @@ export async function generateAPIWorkspaces({
         token = currentToken;
     }
 
+    if (!isCI()) {
+        for (const workspace of project.apiWorkspaces) {
+            for (const generator of workspace.generatorsConfiguration?.groups.flatMap((group) => group.generators) ??
+                []) {
+                if (generator.absolutePathToLocalOutput) {
+                    const { shouldProceed } = await checkOutputDirectory(
+                        generator.absolutePathToLocalOutput,
+                        cliContext
+                    );
+                    if (!shouldProceed) {
+                        cliContext.failAndThrow("Generation cancelled - output directory not empty");
```

Comment:
should be enough to just say `Generation cancelled` since the user opts for that